playbooks/build.yml

```diff
@@ -0,0 +1,108 @@
+---
+- name: Build ceph-ansible image
+  hosts: all
+
+  tasks:
+    - name: Log into registry
+      community.docker.docker_login:
+        registry_url: "{{ docker_registry }}"
+        username: "{{ secret.DOCKER_USERNAME }}"
+        password: "{{ secret.DOCKER_PASSWORD }}"
+      when: push_image | bool
+
+    - name: Run build script
+      ansible.builtin.shell:
+        executable: /bin/bash
+        chdir: "{{ zuul.project.src_dir }}"
+        cmd: |
```

Comment:
I'd prefer to keep the script in its own file, both for readability and in order to be able to run it outside of CI.

Reply:
mYy idea was to pull it 1:1 into Ansible and then rebuild it into real Ansible Tasks.

Reply:
ok, we might want to document how to manually run this, then, but fine for now